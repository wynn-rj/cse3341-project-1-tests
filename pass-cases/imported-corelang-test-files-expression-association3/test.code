program
  int B;
begin
  B := 2 * 3 - 4;
  output B;
end
