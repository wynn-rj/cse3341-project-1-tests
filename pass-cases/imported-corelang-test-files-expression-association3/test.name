expression association3 (Imported from corelan-test-files)
