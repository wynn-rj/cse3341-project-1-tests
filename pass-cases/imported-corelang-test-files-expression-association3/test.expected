program
  int B;
begin
  B:=2*3-4;
  output B;
end
2
